iphone/manifest

```diff
@@ -3,7 +3,7 @@
 # during compilation, packaging, distribution, etc.
 #
 version: 2.0.0
```

Comment:
Thanks, can you please bump the version 2.0.1 ?  and do another module build?

 dist/ti.collectionview-iphone-2.0.1.zip

Reply:
done!